app/api/debug/posts/route.ts

```
@@ -0,0 +1,175 @@
+          }
+        }
+
+        if (issues.length > 0) {
+          frontmatterIssues[filename] = issues;
+        }
+      } catch (error) {
+        frontmatterIssues[filename] = [
+          `Error checking frontmatter: ${error instanceof Error ? error.message : String(error)}`
+        ];
+      }
+    });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)